content/teaching-curriculum/chapter1-high-standards/_index.md

```diff
@@ -2,32 +2,40 @@
 _db_id: 846
 content_type: project
 flavours:
-- none
+  - none
```

Comment:
I changed this back to  "none". The markdown flavour was causing bugs and so it's been completely removed from everything.